sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/EntitysImpl.java

```diff
@@ -24,82 +24,82 @@
 import com.azure.messaging.servicebus.implementation.models.ServiceBusManagementErrorException;
 import reactor.core.publisher.Mono;
 
-/** An instance of this class provides access to all the operations defined in Queues. */
-public final class QueuesImpl {
+/** An instance of this class provides access to all the operations defined in Entitys. */
+public final class EntitysImpl {
```

Comment:
Name change , Shouldn't this be EntitiesImpl  ?

Reply:
This is auto-generated.